web/src/drafts.ts

```diff
@@ -281,7 +281,10 @@ export function rename_stream_recipient(
 }
 
 export function snapshot_message(): LocalStorageDraft | undefined {
-    if (!compose_state.composing() || compose_state.message_content().length <= 2) {
+    if (
+        !compose_state.composing() ||
+        compose_state.message_content().length <= compose_state.MINIMUM_MESSAGE_LENGTH_TO_SAVE_DRAFT
```

Comment:
This could be `!composet_state.has_savable_message_content`, right?

Reply:
Ah yes, thanks!